src/EmbeddedSonarAnalyzer.props

```diff
@@ -9,6 +9,6 @@
     <EmbeddedSonarJSAnalyzerVersion>10.14.0.26080</EmbeddedSonarJSAnalyzerVersion>
     <EmbeddedSonarSecretsJarVersion>2.12.1.2905</EmbeddedSonarSecretsJarVersion>
     <!-- SLOOP: Binaries for SonarLint Out Of Process -->
-    <EmbeddedSloopVersion>10.3.1.78456</EmbeddedSloopVersion>
+    <EmbeddedSloopVersion>10.4.0.78483</EmbeddedSloopVersion>
```

Comment:
It's not a release version

Reply:
https://sonarsource.atlassian.net/browse/SLVS-1234